### PR TITLE
SW-5755 Added review map and replace map in console

### PR DESCRIPTION
**Pull request by tommylau523**

### CODE REVIEW

**Comment by tommylau523** [comment]
* **#2960** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2960?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @tommylau523 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: src/scenes/ApplicationRouter/portal/Map/index.tsx [hunk 1]
```diff
@@ -1,72 +1,15 @@
...
+import { useApplicationData } from 'src/providers/Application/Context';
 import ApplicationPage from 'src/scenes/ApplicationRouter/portal/ApplicationPage';
-import { MapService } from 'src/services';
 import strings from 'src/strings';
-import { MapOptions } from 'src/types/Map';
-
-import { useApplicationData } from '../../../../providers/Application/Context';
-import UpdateOrUploadBoundaryModal from './UpdateOrUploadBoundaryModal';
-
-const MapView = () => {
```
Comment (tommylau523) [inline src/scenes/ApplicationRouter/portal/Map/index.tsx:20]: This class is moved to ApplicationMapCard.tsx

File: src/scenes/ApplicationRouter/portal/MapUpload/index.tsx [hunk 2]
```diff
@@ -1,83 +1,18 @@
...
 import { useLocalization } from 'src/providers';
 import { useApplicationData } from 'src/providers/Application/Context';
-import { requestUploadApplicationBoundary } from 'src/redux/features/application/applicationAsyncThunks';
-import { selectApplicationUploadBoundary } from 'src/redux/features/application/applicationSelectors';
-import { useAppDispatch, useAppSelector } from 'src/redux/store';
 import ApplicationPage from 'src/scenes/ApplicationRouter/portal/ApplicationPage';
 import strings from 'src/strings';
-import useSnackbar from 'src/utils/useSnackbar';
-
-const MapUploadView = () => {
```
Comment (tommylau523) [inline src/scenes/ApplicationRouter/portal/MapUpload/index.tsx:19]: This one is moved to ApplicationMapUploadCard.tsx

File: src/scenes/AcceleratorRouter/Applications/ApplicaitonMapUpload.tsx
Comment (nickgraz) [inline src/scenes/AcceleratorRouter/Applications/ApplicaitonMapUpload.tsx:1]: Typo in the file name

File: src/scenes/ApplicationRouter/portal/Sections/SectionView.tsx [hunk 3]
```diff
@@ -6,8 +6,8 @@ import { useLocalization } from 'src/providers';
 import strings from 'src/strings';
 import { ApplicationDeliverable, ApplicationModule } from 'src/types/Application';
 
+import UpdateOrUploadBoundaryModal from '../../../../components/Application/UpdateOrUploadBoundaryModal';
```
Comment (nickgraz) [inline src/scenes/ApplicationRouter/portal/Sections/SectionView.tsx:9]:
```suggestion
import UpdateOrUploadBoundaryModal from 'src/components/Application/UpdateOrUploadBoundaryModal';
```

**Review by nickgraz — APPROVED** [review]

**Comment by tommylau523** [comment]
### Merge activity

* **Aug 6, 3:09 PM EDT**: @tommylau523 merged this pull request with [Graphite](https://app.graphite.dev/github/pr/terraware/terraware-web/2960).

**Pull request merged**